server/plugins/Twitter.ts

```diff
@@ -5,6 +5,10 @@ import * as TwitterAPI from "twit";
 
 export default class Twitter implements GenericNotifier<{}> {
 
+	public schema: string = `{
+\t_: Boolean
```

Comment:
Don't worry about putting `\t` here or making the indent perfect. I would remove `\t` and just indent everything so it looks nice, GraphQL won't care.